src/helper/onramp.ts

```diff
@@ -61,6 +61,13 @@ export const fetchOnrampSessionToken = async ({
     };
     const res = await fetch(`https://${requestHost}${requestPath}`, options);
 
+    if (!res.ok) {
+      if (res.status >= 500 && res.status < 600) {
+        throw new Error("Server error when requesting token");
+      }
+      return { data: { token: "", error: "Error fetching token request" } };
```

Comment:
This will handle 400 errors. A missing `token` will be reported as a `400` in `route`